### DISCUSSION TITLE
[BUG] Setting the Holiday property to null raises a 400 HTTP error

### DISCUSSION THREAD

**Discussion by rando128** [etsy/open-api#1178] [Q&A]
I can successfully use the [updateListingProperty](https://developers.etsy.com/documentation/reference/#operation/updateListingProperty) endpoint to set values to the Holiday property (id=46803063659)

Nevertheless when I try to delete the property value (by passing   `"values": [null]`), I get a 400 HTTP error despite seeing the value getting properly reset in the Shop Manager.

**Comment by Gareth064** [comment]
So you're saying that you set the values to NULL so you can remove them, and you get a 400 response from the API.
But when you check in the Web App, the values have been removed successfully.

So the bug here is the 400 response.

If so, then please email these details to the Etsy Dev Team at developer@etsy.com and ask them to look into it.

**Comment by rando128** [comment]
Yes that's correct. I'll contact the dev team then.
